CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 0.53.1 (2025-10-02)
+
+
+### Features
+* Add '-p' param support to 'deadline bundle gui-submit' ([`53c507e`](https://github.com/aws-deadline/deadline-cloud/commit/53c507ed817ab9ffde501121515a83677d61db22))
+* Add debug snapshot option to 'deadline bundle submit' ([`c4c96b9`](https://github.com/aws-deadline/deadline-cloud/commit/c4c96b9235fceb96224715a7fc81afa66e2484b5))
+
+### Bug Fixes
+* concurrent job bundle submissions fail due to sqlite3 table already exist (#862) ([`b35aeb1`](https://github.com/aws-deadline/deadline-cloud/commit/b35aeb1eb3ae6955cd4f70f3c2c57f2b0d5bd900))
+* Adding thread local s3 cache db connections to fix performance bottleneck during bundle submission (#848) ([`4914e61`](https://github.com/aws-deadline/deadline-cloud/commit/4914e61f7ede87189bc0debdff36418b6b2796ee))
+* Improved job submission performance by multithreading syscalls to stat() and using manifest file sizes rather than repeating file size check loop a second time (#847) ([`737f781`](https://github.com/aws-deadline/deadline-cloud/commit/737f781712e187ddc2df9d549635333c186b539c))
+* Job submission warning dialogs are too long (#855) ([`6042295`](https://github.com/aws-deadline/deadline-cloud/commit/6042295f758fe03be0b6c69bd242bdc35990de33))
+* List all invalid hidden parameters in GUI validation error (#856) ([`4fa22df`](https://github.com/aws-deadline/deadline-cloud/commit/4fa22df43abc57d053a3f68fef97c8facae12cdc))
+* Remove default requirement and add validation for hidden parameters in GUI (#843) ([`dd01866`](https://github.com/aws-deadline/deadline-cloud/commit/dd0186605715d39c68fd2e42cbf3b5f8c79dd92c))
+* Remove default value requirement for hidden parameters in GUI ([`dd01866`](https://github.com/aws-deadline/deadline-cloud/commit/dd0186605715d39c68fd2e42cbf3b5f8c79dd92c))
+* Add validation for hidden parameters during job bundle loading in GUI ([`dd01866`](https://github.com/aws-deadline/deadline-cloud/commit/dd0186605715d39c68fd2e42cbf3b5f8c79dd92c))
+* Changing wait_for_create_job_to_complete to use a short initial time with an exponential backoff building to a max of 5 seconds in order to speed up job submission times. (#852) ([`bdefa51`](https://github.com/aws-deadline/deadline-cloud/commit/bdefa511d9abe7e741dce10ff56a26b30566cfaa))
+* Handled edge case for re-queued jobs handling sync output cli & some integ test tweaks (#845) ([`a70f0b7`](https://github.com/aws-deadline/deadline-cloud/commit/a70f0b7adeba4434247f2de29cc2dc3453322f27))
+* Handled edge case for re-queued jobs handling sync output cli and some integ test tweaks ([`a70f0b7`](https://github.com/aws-deadline/deadline-cloud/commit/a70f0b7adeba4434247f2de29cc2dc3453322f27))
```

Comment:
Two things:

1.  These look like duplicates
2. I don't understand the impact of the bug from reading this to help readers understand if this fixes their problem. Looking at the linked PR description it says:

    >   We found an unhandled edge case with the integ tests for the sync-output a.k.a incremental download feature. In case of re-queued tasks sometimes the CLI ends up downloading the same output more than once due to the eventual consistency window if the tasks get executed and re-queued/re-executed within the same 2 min window.


    What about instead we use:

    >   sync-output downloads duplicate files when tasks requeued


    maybe we can sync with the author or reviewers to confirm